demos-europe/demosplan-core
# feat (DPLAN-3796) Fix bulk segement editor test

Description:
- Make work again SegmentBulkEditorServiceTest
- Rename file to SegmentBulkEditorServiceTest

### How to review/test
- Run the test in local, all tests should pass

Delete the checkbox if it doesn't apply/isn't necessary.
